### PR TITLE
Add some basic website resources to docs

### PR DESCRIPTION
**Pull request by rwy7**
Slightly rearrange the directories, add some preliminary jekyll configurations, and move some text from SetupBase9.md to index.md.

In the DeveloperJourney.md, autogenerate the table of contents, and add language tags to the code blocks.

Move the presentation pdfs to presentations.

Signed-off-by: Robert Young <rwy0717@gmail.com>

### CODE REVIEW

File: docs/SetupBase9.md [hunk 1]
```diff
@@ -1,16 +1,4 @@
...
-
-Base9 is a tiny virtual machine and programming language! We’re using it to show people how to use [Eclipse OMR] to build their own programming language. 
-
-[Eclipse OMR]: https://github.com/eclipse/omr
-
-The Base9 language is a subset of JavaScript and is very limited in functionality. It currently only contains support for integers. 
-
-For a full overview of Base9 and it's components, plus a tutorial on adding new bytecodes, see our [Supercharge a Language Runtime] presentation and tutorial. 
-
-[Supercharge a Language Runtime]: ./CASCON2017_SuperchargeALanguageRuntime.pdf
```
Comment (arianneb) [inline docs/SetupBase9.md:13]: JW why you removed this? @rwy0717

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Base9 is a tiny virtual machine and programming language! We’re using it to show people how to use [Eclipse OMR] to build their own programming language. 
+
+[Eclipse OMR]: https://github.com/eclipse/omr
+
+The Base9 language is a subset of JavaScript and is very limited in functionality. It currently only contains support for integers. 
+
+For a full overview of Base9 and it's components, plus a tutorial on adding new bytecodes, see our [Supercharge a Language Runtime] presentation and tutorial. 
+
+[Supercharge a Language Runtime]: ./presentations/CASCON2017_SuperchargeALanguageRuntime.pdf
```
Comment (arianneb) [inline docs/index.md:13]: I think we should link to the Developer Journey and the Setup Page from here. This is going to be the websites main landing page right? @rwy0717

File: docs/SetupBase9.md (hunk 1 above)
Comment (rwy7) [reply docs/SetupBase9.md:13]: I moved it to the index--It looked more like an introduction, so I thought it'd be a good starting point for our homepage.

File: docs/index.md (hunk 2 above)
Comment (rwy7) [reply docs/index.md:13]: 100% agree. I haven't written any new content, just pulled in some of the setup & intro stuff to the index. That would be my very next PR, unless you want me to just do it now 👍

**Pull request merged**